se2_test.go

```diff
@@ -163,7 +164,7 @@ func TestGetAndExec(t *testing.T) {
 
 	// Tests the administrative results endpoint
 	t.Run("ExecResultMetadata", func(t *testing.T) {
-		res, err := client.FunctionResultMetadata(uuid)
+		res, err := client.ResultMetadata(uuid)
```

Comment:
`ResultMetadata` or `ExecutionResultMetadata` We should match with the node one as well.